src/install/ai-monitoring/agent-lang/node-config.mdx

```diff
@@ -0,0 +1,104 @@
+---
+componentType: default
+headingText: Configure the Node.js agent
+freshnessValidatedDate: never
+---
+
+To collect AI data, you need to ensure that certain configurations are enabled when you instrument your AI-powered app. We also recommend updating other variables to improve data retention.
+
+* We recommend applying these configurations in `newrelic.js`. 
+* If you have a more complex set up with multiple environments, you can configure with environment variables.
+
+1. Set these variables:
+
+    <table>
+        <thead>
+            <tr>
+                <th>
+                    Configuration file
+                </th>
+                <th>
+                    Environment variable
+                </th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>
+                    `ai_monitoring.enabled = true`
+                </td>
+                <td>
+                    `NEW_RELIC_AI_MONITORING_ENABLED = TRUE`
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    `distributed_tracing.enabled = true`
+                </td>
+                <td>
+                    `NEW_RELIC_DISTRIBUTED_TRACING_ENABLED = TRUE`
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    `high_security = false`
```

Comment:
High security mode is a handshake configuration. Setting it to false in configuration does not mean it's not enabled on the account level as a server side configuration.  We probably just want to call out that if your account has high security mode enabled it, AIM will not function

Reply:
Here's more info on this feature https://docs.newrelic.com/docs/apm/agents/manage-apm-agents/configuration/high-security-mode/

Reply:
after a tech writing squad discussion, we decided to remove any specific configurations for high security mode in the install path